index.html

```diff
@@ -52,22 +52,31 @@ <h1 class="masthead-heading mb-0">GEOSTAC</h1>
         </div>
     </header>
     <section class="page-section  text-black mb-0 bg-secondary" id="problem">
-        <div style="margin: 0px auto; position: relative; overflow: hidden;
-        padding: 1em" >
-          <div id="content">
-            <div>
               <p class= "usgs">
                 <img src="assets/USGS_logo_green_SQUARE.png" alt="">
               </p>
-              <h2 class="overview">
+              <h2 class="overview" style="text-align:Center;">
                 Overview of Our Project:
               </h2>
               <p class="overview_paragraph">
                   This project focuses on the expansion of a Leaflet Map Viewer (CartoCosmos) which provides a modular map view of our planets. Our expansion focuses on displaying individual planetary images footprints based on the images location on a particular planet. These images are provided from SpatioTemporal Asset Catalogs (STAC). The next scope of this project is to provide the ability for the user to explore, click, and load the selected images within the Leaflet Viewer. This will provide the user the ability to pull and use large TIFF format images without the need to download or process the images. This is beneficial to the planetary science community to focus more on their scientific research of our planets.
               </p>
-            </div>
-        </div>
-      </div>
+              <br>
+              <br>
+              <br>
+              <h3 class="overview" style="text-align:Center;">
+                Learn more about our project....
+              </h3>
+              <div class="divider-custom divider-secondary">
+                <div class="divider-custom-line"></div>
+                <div class="divider-custom-icon"><i class="fas fa-star"></i></div>
+                <div class="divider-custom-line"></div>
+              </div>
+              <center>
+                <iframe width="420" height="315"
+                src="https://www.youtube.com/watch?v=Tv9mCg4ejCM">
```

Comment:
Video is now public on YouTube. May be a different link now.

Reply:
this should be the link: https://youtu.be/Tv9mCg4ejCM

Reply:
This is the code to use. You have to click share on YouTube then click embed to HTML.
`<iframe width="560" height="315" src="https://www.youtube.com/embed/Tv9mCg4ejCM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>`